src/tcgwars/logic/impl/gen3/PowerKeepers.groovy

```diff
@@ -595,7 +615,7 @@ public enum PowerKeepers implements LogicCardInfo {
           text "Once during your opponent's turn, when any of your Pokémon is Knocked Out by your opponent's attacks, you may use this power. Choose a basic Energy card discarded from the Knocked Out Pokémon and attach it to Lanturn. You can't use more than 1 Energy Grounding Poké-Power each turn."
           delayedA{
             before KNOCKOUT, {
-              if ((ef as Knockout).byDamageFromAttack && bg.currentTurn==self.owner.opposite && ef.pokemonToBeKnockedOut!=self  && ef.pokemonToBeKnockedOut.cards.energyCount(C)) {
+              if ((ef as Knockout).byDamageFromAttack && bg.currentTurn==self.owner.opposite && ef.pokemonToBeKnockedOut != self  && ef.pokemonToBeKnockedOut.cards.filterByType(BASIC_ENERGY).energyCount(C)) {
                 if (confirm("Move an energy from ${ef.pokemonToBeKnockedOut} to $self?")) {
                   moveEnergy(basic:true, ef.pokemonToBeKnockedOut,self)
                 }
```

Comment:
Does this handle the "Once during your opponent's turn" part of the ability properly?

Reply:
> bg.currentTurn==self.owner.opposite

That should cover for it, and there should be no way for a KO to be caused by damaged during between turns.

Reply:
An attack that knocks out the active and a benched Pokémon or multiple benched Pokémon, such as Tag Bolt-GX or a spread attack would activate this multiple times from my understanding.

Reply:
Hmm, that's a potential case. Hadn't noticed it had that difference with the """reprint""" from Celestial Storm.

Compendium doesn't seem to cover that case, the Muk-ex one is potentially a timing nightmare though oof.

Reply:
Ok Muk-ex was only in a forum post, must've been removed after confirming.

Back to this, the "once per turn" probably cover most potential cases. Will add the restriction in #877